### ISSUE TITLE
Phone Interactions

### ISSUE THREAD

**Issue by TheRealMarioBro** [Monika-After-Story/MonikaModDev#5559]
I had an idea where Monika could interact with your phone in some way. I don't know how, maybe... something simple like sending little messages to you during the day. Maybe saying she misses you when you're gone for a while. You could probably tell her to remind you of something. I think it'd be cool.

**Comment by U6Luna** [comment]
Maybe she could also become like one of those assistant things you can get to help you with things on the phone, and access the microphone.

**Comment by DoodleSauce** [comment]
i mean dont forget its a mod in ddlc guidelines

**Comment by DoodleSauce** [comment]
MAYBE it'll violate some laws

**Comment by multimokia** [comment]
Not really feasible as you start to delve into costs administered by services which allow you to send text messages programmatically, additionally with some getting them from their carriers. Texting is definitely out of the question. 

Additionally, you'd have to leave MAS running on your computer whenever you go out in order for her to text you, and I know a lot of people like to take her with them when going out. Seems like it won't really add that much for how much it'd take to develop timewise and costwise.

**Comment by DoodleSauce** [comment]
i read this from the guidelines 

> You are NOT allowed to redistribute DDLC or fan games on any app store including Steam, the Google Play Store, the Apple App Store, the Windows Store, GameJolt, itch.io, or any others. You may not port DDLC or any fan game to these platforms.<

**Comment by TheRealMarioBro** [comment]
Well maybe she can just send you something like a file when you plug in your phone. Maybe she can send you a game. Or you can send her stuff like images.

**Comment by U6Luna** [comment]
That actually sounds pretty cool. Like when you upload her over it also uploads a few other files.

**Comment by multimokia** [comment]
Detecting what's a phone and not a normal drive/peripheral is basically impossible as getting the information is difficult. Doing so would also likely be a cross platform nightmare, especially looking at Apple devices here, given that you basically ***need*** iTunes to interface with any of the data on the device in the first place.

Adding random files, let alone installing random apps to devices is an awful practice and is extremely intrusive over a user's items. It's basically giving people bloatware.

If we controlled the applications added, that's essentially asking us to become mobile devs to ship a specific app. What's the point?

**Comment by TheRealMarioBro** [comment]
This whole phone thing is a suggestion of mine, I'm not demanding it. I don't care if it is or isn't in the game.

**Comment by multimokia** [comment]
I know it's a suggestion. I'm just giving you reasons where the suggestion is flawed and where it won't work.

**Comment by TheRealMarioBro** [comment]
I see. I must've misjudged your character. Well thank you for telling me.

**Comment by DoodleSauce** [comment]
its kinda sad cause at first it seemed like a super nice update but ye

Reality is harsh

**Comment by ThePotatoGuy** [comment]
in the past we've been against sms because of the server overhead, but I might reconsider as I have an extra server doing nothing right now. It realistically depends on sms costs so if its still financially a problem then we won't pursue it.

**Comment by DoodleSauce** [comment]
thanks ThePotatoGuy always here to make this mod better :D

i honnestly hope you guys can achieve the required amount of money

btw does this mod has a patreon or anything like that ?

**Comment by Dank-Elf-Boy** [comment]
@TheRealMarioBro It sounds like you're referring to that Gatebox thing from Japan lol.

@ThePotatoGuy I have no idea how MAS will be able to send messages to each user's phones but I found a website that mentions avoiding SMS cost.
https://www.howtogeek.com/164395/how-to-eliminate-sms-fees-and-text-for-free/

It'd be cool to be able to communicate with Monika where ever you go, but what if MAS can't keep track of your conversations? If I were able to have a conversation with her via text, I'd at least want her to remember that conversation when I came back to the game. If this feature is guaranteed then I'll be all for it. Either way, it sounds complicated lol.
